braindecode/datasets/base.py

```diff
@@ -142,6 +144,88 @@ def transform(self, value):
         self._transform = value
 
 
+class SequenceDataset(WindowsDataset):
```

Comment:
@hubertjb @robintibor I still wonder whether functionalities of this class couldn't be added to the `WindowsDataset`. It would need parameters in `WindowsDataset.__init__` that specify the `seq_len=1` and `step_len=1` with default values and the rest of the behavior would stay similar (we have to keep `self.start_inds` for WindowsDataset but with default parameters it should behave the same as before)? I would say that the behavior of this class is not so different from the base class. As well `label_transform` may be used in the standard case without multiple windows.

Reply:
In any case maybe a better name would be `SequenceWindowsDataset`?

Reply:
Yes I agree that SequenceWindowsDataset is maybe more explicit